pyproject.toml

```diff
@@ -11,6 +11,7 @@ dependencies = [
 
 [tool.commitizen]
 bump_message = "bump: v$current_version → v$new_version"
+hooks_pre_bump = ["uv lock --offline"]
```

Comment:
Shouldn't this be `hooks_post_bump`? AFAIK pre-bump means "before incrementing the version", and post's meaning follows from that. The re-locking only makes sense after the version number is incremented.

Reply:
The docs describe the pre bump hooks as "A list of optional commands that will run right after updating version_files and before actual committing and tagging the release.", which sounds like it's what we need, or not?

Reply:
Checked the docs and nevermind, your implementation is correct!